src/Lesson1/Task1/ArrayExcerciseVariantOne.java

```diff
@@ -0,0 +1,83 @@
+package Lesson1.Task1;
+
+/*
+* 1 Написать программу, осуществляющую обработку массива. Исходные условия:
+массив содержит только целые числа от -10 до 10 (допускается генерация
+элементов с помощью метода (int) Math.random()), программа должна выводить
+в консоль исходный массив и полученный результат, количество элементов в
+массиве 20.
+Варианты задания:
+1. В массиве целых чисел поменять местами максимальный отрицательный
+элемент и минимальный положительный.*/
+
+/**
+ * @author Andrii Prokofiev
+ * @version 1.0
+ */
+
+public class ArrayExcerciseVariantOne {
+
+    public static void main(String[] args) {
+        ArrayExcerciseVariantOne arrayExcercises = new ArrayExcerciseVariantOne();
+        arrayExcercises.startProgram();
+    }
+
+    public void startProgram() {
+        int[] arr = getIntArray();
+        int minNegativeIndex = getMaxNegativeIndex(arr);
+        int minPositiveIndex = getMinPositive(arr);
+        System.out.println("Index with min neg = " + minNegativeIndex);
+        System.out.println("Index with min pos = " + minPositiveIndex);
+        for (int i = 0; i < arr.length; i++) {
+            System.out.println("[" + i + "] " + arr[i]);
+        }
+
+        int temp = arr[minNegativeIndex];
+        arr[minNegativeIndex] = arr[minPositiveIndex];
+        arr[minPositiveIndex] = temp;
+
+        for (int i = 0; i < arr.length; i++) {
+            System.out.println("[" + i + "] " + arr[i]);
+        }
+
+
+    }
+
+    private static int getMaxNegativeIndex(int[] array) {
+        int min = array[0];
+        int minIndex = -1;
```

Comment:
You set first array value to 'min' value, and later one in the cycle you again set first value to same variable. You should do it only once. There are 2 options:

1. At line 47 set min to 0;
2. At line 48 start index not from -1 but from 0;